dart/lib/src/scope_extensions/_io_scope_extensions.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:io';
+
+import '../../attachment.dart';
+import '../scope.dart';
+
+extension IoScopeExtensions on Scope {
+  Future<void> addAttachementPath(
```

Comment:
Actually I think `Attachment` should have a couple of factory constructors.
- `Attachment.fromPath()`
- `Attachment.fromFile()`

and so on